pkgs/development/typst-packages/polylux.nix

```
@@ -0,0 +1,23 @@
+{
+  lib,
+  buildTypstPackage,
+  fetchFromGitHub,
+}:
+
+buildTypstPackage rec {
+  pname = "polylux";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "andreasKroepelin";
+    repo = "polylux";
+    rev = "v${version}";
+    hash = "sha256-jErxl2s2xez2huUwpsT6N1pZANvuZMdIt4taFOurCtU=";
+  };
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
The same goes for all other `meta` attributes. Nesting `with` expressions is considered a bad practice.